Formula/g/git-xet.rb

```diff
@@ -0,0 +1,31 @@
+class GitXet < Formula
+  desc "Git LFS plugin that uploads and downloads using the Xet protocol"
+  homepage "https://github.com/huggingface/xet-core"
+  url "https://github.com/huggingface/xet-core/archive/refs/tags/git-xet-v0.2.0.tar.gz"
+  sha256 "551237dbed960265804c745d50531cc16d8f13fb31e1900c9c6e081bfab01874"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_tags
+    regex(/^git-xet-v(\d+(?:\.\d+)+)$/i)
```

Comment:
For any small variation, we prefer to use 
```suggestion
    regex(/^git[._-]xet[._-]v?(\d+(?:\.\d+)+)$/i)
```